scipy/linalg/tests/test_build.py

```diff
@@ -19,7 +19,7 @@ def __init__(self):
         self.cmd = ['ldd']
 
         try:
-            st = call(self.cmd, stdout=PIPE, stderr=PIPE)
+            _ = call(self.cmd, stdout=PIPE, stderr=PIPE)
```

Comment:
rather than `_ =` you can just leave out assignment completely. `_` is more useful in cases of `used, unused = somefunc()`

Reply:
Done for all examples changed.